framework/index.js

```diff
@@ -28,6 +28,7 @@ module.exports = {
 	Queue: require('./src/queue'),
 	mapper: require('./src/mapper'),
 	MySQL: require('./src/mysql'),
+	MySQLKVStore: require('./src/mysqlKVStore'),
```

Comment:
```suggestion
	MySQL: { ...require('./src/mysql'), KVStore: ...require('./src/mysqlKVStore') },
```

Reply:
Did the above changes in https://github.com/LiskHQ/lisk-service/blob/7600300c23c78c5f749aeb8959c61df09e947d03/framework/src/mysql/index.js